tests/Functions.php

```diff
@@ -0,0 +1,81 @@
+<?php
+require_once '../functions.php'; // Require the file to be tested
+
+use PHPUnit\Framework\TestCase; // Need this to use PHPUnit
+
+class Functions extends TestCase
+{
+    // Success Tests
+    public function test_displayCollection_GivenArrayReturnsString()
+    {
+        // Arrange
+        $mockDatabaseResult =
+            [
+                [
+                    'discipline' => 'road',
+                    'name' => 'test',
+                    'region' => 'test',
+                    'country_name' => 'test',
+                    'url' => 'www.test.com',
+                    'alt_text' => 'test',
+                    'short_description' => 'testing testing testing',
+                    'distance' => 123,
+                    'effort_level' => 6,
+                    'adrenaline_rating' =>6
+
+                ]
+            ];
+
+        $expected =
+            '<div class="card">'
+            .'<div class="card-header">'
```

Comment:
Space between concatenation operator `.` and the next thing (in this case quote `'`).

Reply:
done